interpreter/cling/include/cling/Interpreter/Interpreter.h

```diff
@@ -281,7 +281,22 @@ namespace cling {
     ///\param[in] argv - arguments passed when driver is invoked.
     ///\param[in] llvmdir - ???
     ///\param[in] noRuntime - flag to control the presence of runtime universe
-    Interpreter(int argc, const char* const *argv, const char* llvmdir = 0, bool noRuntime = false);
+    ///\param[in] isChildInterp - flag to control if this is a child interpreter or not.
+    ///
+    Interpreter(int argc, const char* const *argv,
+                const char* llvmdir = 0, bool noRuntime = false, bool isChildInterpreter = false);
```

Comment:
I left this for the end. I guess that with the use of the private function I will replace the constructor delegation , right?